src/aatoolbox/datasources/codab.py

```diff
@@ -0,0 +1,76 @@
+"""Retrieve COD administrative boundaries."""
+from pathlib import Path
+
+import geopandas as gpd
+
+from aatoolbox.datasources.datasource import DataSource, check_file_existence
+from aatoolbox.utils.hdx_api import get_dataset_from_hdx
+
+MODULE_BASE = "cod_ab"
```

Comment:
Similar question here as above, is it best practice to assign `MODULE_BASE` external to the class? Are users supposed to be able to change it? Or is this just again because it's easier for us to change `MODULE_BASE` at the top of the script if we decide to change the folder name in the future or something?

Reply:
Same as before, it's the [PEP 8 convention](https://www.python.org/dev/peps/pep-0008/#constants) for constants. Although in this case I'm not super happy with the name. Basically it's like the base name for the directories and files.